### ISSUE TITLE
Does not work at all

### ISSUE THREAD

**Issue by Saddamus** [niklabh/mattermost-plugin-webrtc-video#32]
OS: Mac and Linux (Fedora)
Browser: Chrome
- Does not ask for camera or microphone permissions
- Does not send any packets to defined stun and turn address (wireshark).
- Call window shows connecting, but nothing pops up on the other side. 
Would you like to help me to debug this issue?
![obraz](https://user-images.githubusercontent.com/7191172/82145253-c26a4380-9849-11ea-9695-dac8201227d5.png)

**Comment by RosiersRobin** [comment]
I have the same issues, the thing is, I get the TURN/STUN servers working correctly, and have a decent signalhub, on which I can see the calls comming.


The modal just states "connecting"??

**Comment by Saddamus** [comment]
Yes, it shows connecting. I have deployed signalhub instance on the same server with ssl cert, i can see subscriptions and call events on logs of this service. I have also configured to my own turn / stun service, which is already used on production for other webrtc deployment.
My previous statement regarding stun and turn was incorrect, because i have had captured WiFi interface, while routing was being sent over VPN, so with and without VPN i can see succesful turn and stun dialog / bindings, but no chenge regarding calls. I dont understand the flow, so dont know how to debug. So far i guess, users subscribe to signalhub where they should be notified by events, that someobody is calling them, but how does it keep subscription alive, over web socket ?

**Comment by sebastieniko** [comment]
I'm not able to make it  too.. Is this plugin working?
Is there a log file somewhere to get some errors messages? 
I installed the servers as explained in the doc, but it cant connect with mattermost 5.25.2

**Comment by MartB** [comment]
Can confirm the "connecting" the signal hub server seems to have problems.

**Comment by gustavotrott** [comment]
I tried to install the plugin but also had problems while "connecting"..
I had to disable the plugin unfortunately.

**Comment by GP-S** [comment]
Downgrading to 1.0.2 seems to partially fix the issue (no more null state) but still stuck on connecting modal. We can provide a test instance if needed.

**Comment by gpopesc** [comment]
the same issue here, it is not working, blocked in connecting

**Comment by crisicon** [comment]
Does anyone know how to solve this issue?
I can see traffic to signalhub
No packets being sent to stun and turn servers
Both windows app and web browser instance sitting in "connecting"

**Comment by niklabh** [comment]
Fixed
